corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -107,8 +107,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
     var showMenu = function (menuResponse) {
         var menuListView = menusUtils.getMenuView(menuResponse);
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
-        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview;
-
+        var queryResponse = menuResponse.queryResponse;
+        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview && queryResponse && queryResponse.displays.length;
         if (sidebarEnabled && menuResponse.type === "query") {
```

Comment:
I think this is the area related to the[ bug QA found ](https://dimagi-dev.atlassian.net/browse/QA-5945). When when `sidebarEnabled` is false, the region shown is `menuListView`. For a `query` type `menuResponse`, `menuListView` would be the regular `queryView`

Reply:
Thanks @Jtang-1, this was something that Steph brought up in an earlier comment. I'd made the fix already, but it hadn't been deployed yet when the QA bug popped up. I deployed and tested this morning and things look good.